opal/mca/pmix/pmix112/pmix/include/Makefile.am

```diff
@@ -25,7 +27,8 @@ headers += \
         include/private/pmix_socket_errno.h \
         include/private/pmix_stdint.h \
         include/private/prefetch.h \
-        include/private/types.h \
+        include/private/types.h
+nodist_headers += \
```

Comment:
That looks more elegant at first glance.
I will not be able to test and update the pr before tomorrow though.